extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaAnonWarningWidget.scss

```diff
@@ -0,0 +1,19 @@
+$padding-top-bottom: 8px;
+$padding-left-right: 13px;
+.ve-ui-wikia-anon-warning {
+	background-color: rgba($wds-color-warning, .8);
+	color: $color-button-text;
+	font-size: 14px;
+	height: 35px - 2 * $padding-top-bottom;
+	margin: 10px 0;
+	padding: $padding-top-bottom $padding-left-right;
+	width: calc(100% - 26px);
+
+	a {
```

Comment:
As far as I remember Inez presentation few years ago, styling using tag names is forbidden in VE ;)